defaults/settings.yml.default

```diff
@@ -43,3 +43,8 @@ transmissionvpn:
   vpn_endpoint: Netherlands.ovpn
 unifi:
   port: 8080
+xbackbone:
+ domain:
```

Comment:
Do we need to bother asking to fill out the domain & possibly even subdomain? Domain in the role should just be pulled from the accounts.yml domain and for subdomain should it not just be xbackbone? Or a shorter version of it instead of giving the user a choice?

Reply:
I wanted to be able to choose a different domain to access this from.  Simliar to the Jirafeau role.

I also didn't want the URL's to be xbackbone.domain.tld, so I added the option to choose your subdomain.    i.domain.tld is better IMO.  (similar to imgur links when uploading via ShareX)

Reply:
Makes sense, no need to change these then